R/doeResponseSurfaceMethodology.R

```diff
@@ -101,6 +125,13 @@ doeResponseSurfaceMethodology <- function(jaspResults, dataset, options, ...){
       }
     } else {
       for (i in 1:(options[["numberOfFactors"]])) {
+        if (!(is.na(as.numeric(options[["factors"]][[i]][["factorName"]])))) {
+          text <- gettext(paste0("Factor name '",options[["factors"]][[i]][["factorName"]],
+                                 "' is a number, and number factor names are not supported."))
```

Comment:
You can only put string literals inside `gettext` or `gettextf`. So never do `gettext(paste(...))`, but instead do `gettextf("Factor name '%s' ...", stringToInsert)`. See `?sprintf` for details, and make sure to use `%1$s` and `%2$s` to indicate multiple arguments.

Please fix this everywhere.